website/app.js

```diff
@@ -3,35 +3,65 @@ const apiEndPoint = "api.openweathermap.org";
 const apiKey = "081a059f88749d9d3c5a61e73985ef20";
 const country = "us";
 
-// Create a new date instance dynamically with JS
-let d = new Date();
```

Comment:
**d** is not a good name, also **let** means that it will be overriden and this variable won't, so better would be to have a **const** declaration.